themes/digital.gov/layouts/topics/list.html

```diff
@@ -3,11 +3,250 @@
 */}}
 
 {{- define "content" -}}
-  <main role="main" id="main-content" class="main-topic">
-    {{ if .Params.topic_collection }}
-      {{ partial "core/topics/topics-collection.html" . }}
+  {{- $resources := (where .Pages "Section" "resources") -}}
+  {{- $services := (where .Pages "Section" "services") -}}
+  {{- $communities := (where .Pages "Section" "communities") -}}
+  {{- $events := (where .Pages.ByDate.Reverse "Section" "events") -}}
+  {{- $posts := (where .Pages "Section" "news") -}}
+  {{/* Use pagination query for news and event streams */}}
+  {{- $events_paginated := (where .Paginator.Pages.ByDate.Reverse "Section" "events") -}}
+  {{- $past_events_paginated := (where $events_paginated ".Date.Unix" "<" now.Unix) -}}
+  {{- $past_events_paginated = (where $events_paginated ".Params.youtube_id" "!=" nil) -}}
+  {{- $posts_paginated := (where .Paginator.Pages "Section" "news") -}}
+  {{- $stream := union $posts_paginated $past_events_paginated -}}
+  {{- $stream = $stream.ByDate.Reverse -}}
```

Comment:
DRY'ed up data queries to be called once instead of multiple times and moved to top of page